chromium/browser_type.go

```diff
@@ -399,7 +378,9 @@ func execute(ctx context.Context, path string, args, env []string, cleanupFunc f
 		return nil, nil, fmt.Errorf("context err: %w", ctx.Err())
 	}
 	go func() {
-		defer cleanupFunc()
+		defer func() {
+			_ = dataDir.Cleanup() // Log when possible
+		}()
```

Comment:
Should we log here?

Reply:
Resolved in commit: 15022675046f46849db9e5f3c3ae3f54b1683cfa